docs-ref-conceptual/install-azure-cli-windows.md

```diff
@@ -97,6 +97,36 @@ You can also use `winget`, Microsoft's Package manager for Windows, to install a
 
 The `-e` option is to ensure the official Azure CLI package is installed. This command installs the latest version by default. To specify a version, add a `-v <version>` with your desired version to the command.
 
+# [ZIP Package](#tab/zip)
+
+> [!IMPORTANT]
+> This package is currently in preview.
+
+You can use the ZIP package to install the Azure CLI on Windows. This package is useful when you don't have administrative privilege.
+
+Unzip the package to a folder, then using Cmd or PowerShell, call the Azure CLI by running `<unzipped folder path>\bin\az.cmd`.
+
+If you want to run the `az` command directly, then follow the steps below to add the Azure CLI to your PATH environment variable.
+1. Open the Start Menu and search for `environment variables`.  Click `Edit the system environment variables`.
+1. Click the `Environment Variables...` button.
+1. In the `User varibles for <username>` box, select `Path` and click the `Edit...` button.
+1. Click the `New` button and add the `<unzipped folder path>\bin` to the list.
+1. **Restart your terminal.**  You should now be able to run `az` commands.
+
+### Latest version
+
+> [!div class="nextstepaction"]
+> [Latest release of the Azure CLI (64-bit)](https://aka.ms/installazurecliwindowszipx64)
```

Comment:
The button title will be the same as MSI:

![image](https://github.com/MicrosoftDocs/azure-docs-cli/assets/4003950/4ca7900f-7494-4479-a85e-c0dd264e3eb9)

How about using

- Latest MSI of the Azure CLI (32-bit)
- Latest MSI of the Azure CLI (64-bit)
- Latest ZIP of the Azure CLI (64-bit)